test/schema.test.ts

```diff
@@ -23,7 +25,9 @@ describe("commands/schema", function () {
 
   test("throws with invalid chain", async function () {
     const consoleError = spy(console, "error");
-    await yargs(["schema", "valid_9999_0"]).command(mod).parse();
+    await yargs(["schema", "valid_9999_0", "--chain", "local-tabd"])
+      .command(mod)
+      .parse();
```

Comment:
So this is now testing the --chain flag, not the chain id in the table name. Maybe that is fine now since the sdk should deal with testing the chain id in the table name.

Reply:
That's right, so actually I'm not sure why we need this test anytmore if that's the case. Ideally we'll still check that the cli is till pikcing up the right table name

Reply:
I've updated this, it is now testing the chain in the tableland.